### ISSUE TITLE
using this on unraid

### ISSUE THREAD

**Issue by pcgirl** [jishi/node-sonos-http-api#573]
I have loaded the docker from here https://github.com/chrisns/docker-node-sonos-http-api and it seems to work good except that I can't seem to get the presets working.  I have tried making the directories as per his instructions but it doesn't seem to make any difference.  It still comes back an error.  I am wondering if your would have any idea how to get this working?

**Comment by jishi** [comment]
Do you still have a problem with this?

**Comment by pcgirl** [comment]
Yes, it still won't let me use Ryan's Bedroom but it will let me use Lauren's Bedroom.


Thanks,
Jane Johnson



________________________________

**Comment by dwightstanley** [comment]
Not sure about your single player issue. This is how I 'altered' the Unraid Docker Settings to make it easier for Presets, etc.
![sonos-basic settings](https://user-images.githubusercontent.com/25987898/37222042-db96e4d4-2399-11e8-90de-0ff6c1bbe134.JPG)
This is how I 'added' the folders so that they are accessible under the Unraid Appdata 'sonos' folder. Much easier to find and they won't get erased by container updates.
![sonos-advanced settings](https://user-images.githubusercontent.com/25987898/37222122-0d155306-239a-11e8-9f3a-27f5bcf074c6.JPG)
I also added the line:
`https://github.com/jishi/node-sonos-http-api/blob/master/static/sonos-icon.png?raw=true`
in the advanced settings (upper right toggle) so that the docker has an icon in Unraid.

**Comment by pcgirl** [comment]
Hey thanks for the update on the settings.  I had actually figured this out but I didn't post it.  Should have I guess.  Thanks for the icon URL though.


Thanks,
Jane Johnson
++++++++++++++++++
A+ Computer Services
(905) 317-0540
++++++++++++++++++


________________________________
